powa_collector/__init__.py

```diff
@@ -188,15 +198,17 @@ def main(self):
             self.stop_all_workers()
 
     def register_worker(self, name, repository, config):
-            self.workers[name] = PowaThread(name, repository, config)
-            # self.workers[s].daemon = True
-            self.workers[name].start()
+        """Add a worker thread to a database"""
+        self.workers[name] = PowaThread(name, repository, config)
+        self.workers[name].start()
 
     def stop_all_workers(self):
+        """Ask all worker threads to stop"""
```

Comment:
should we document that this is an async notification that should be processed hopefully soon, but without much guarantee?